inspec.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'thor', '~> 0.20'
   spec.add_dependency 'json', '>= 1.8', '< 3.0'
   spec.add_dependency 'method_source', '~> 0.8'
-  spec.add_dependency 'rubyzip', '~> 1.1'
+  spec.add_dependency 'rubyzip', '~> 1.2.2'
```

Comment:
This tightens the constraint from major-level (minor upgrades allowed, major held) to minor level (patch upgrades allowed, minor held).  

Consider '~> 1.2, >= 1.2.2' (minor upgrades allowed starting from 1.2, 1.2.2 or newer)

Reply:
Ok good call. I'll update this.